test/specs/route-createRules.js

```diff
@@ -0,0 +1,41 @@
+
+describe('PushStateTree createRule', function() {
+  'use strict';
+
+  describe('when rule /^servers(\/)?(.*)/', function() {
+    var pst;
+    var rule;
+    beforeEach(function() {
+      pst = new PushStateTree();
+      rule = pst.createRule({
+        id: 'servers',
+        rule: /^servers(\/)?(.*)/
```

Comment:
you can expose the object with id and rule, to compare results inside of the tests.